luigi/cmdline_parser.py

```diff
@@ -61,15 +60,14 @@ def __init__(self, cmdline_args):
         """
         Initialize cmd line args
         """
-        self.cmdline_args = cmdline_args
         known_args, _ = self._build_parser().parse_known_args(args=cmdline_args)
         self._attempt_load_module(known_args)
+        known_args, _ = self._build_parser().parse_known_args(args=cmdline_args)
```

Comment:
Add a comment in the code. The `test_bin_luigi_options_before_task` would break if you delete this line.